isites_migration/jobs.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from async.models import Process
+
+from isites_migration.utils import export_files, import_files
+
+
+logger = logging.getLogger(__name__)
+
+
+def migrate_files(keyword, canvas_course_id, process_id=None):
+    logger.info("Starting migrate_files job for keyword %s and canvas_course_id %s", keyword, canvas_course_id)
+
+    process = Process.objects.get(id=process_id)
```

Comment:
On second thought, I reworked this a bit so that process_id is a required parameter.